package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "ts-jest",
   "version": "23.0.1",
-  "main": "index.js",
+  "main": "dist/index.js",
   "types": "./dist/index.d.ts",
   "description": "A preprocessor with sourcemap support to help use Typescript with Jest",
   "scripts": {
-    "build": "cpx index.d.ts dist/ && tsc -p .",
-    "build:watch": "cpx index.d.ts dist/ && tsc -p . -w",
-    "test:nolint": "npm run clean-build && node scripts/tests.js",
-    "clean": "rimraf dist/**/* && rimraf tests/simple/coverage && rimraf tests/simple-async/coverage && rimraf tests/**/*/debug.txt && rimraf tests/**/node_modules",
+    "build": "tsc -p tsconfig.build.json",
+    "build:watch": "tsc -p tsconfig.build.json -w",
+    "test:nolint": "npm run clean-build && jest --clearCache && node scripts/tests.js",
+    "clean": "rimraf dist/**/* && rimraf tests/*/coverage && rimraf tests/*/debug.txt && rimraf tests/*/node_modules",
     "clean-build": "npm run clean && npm run build",
     "pretest": "npm run tslint && npm run clean-build",
     "test": "node scripts/tests.js",
-    "tslint": "tslint src/*.ts",
+    "test:unit": "jest --testRegex '' --testMatch '<rootDir>/src/**/*.spec.ts'",
```

Comment:
I don't think this will correctly update results if the ts-jest sources are updated, only if the test sources are. What's it for exactly?

Reply:
You cannot use `testRegex` alongside `testMatch`, so I had to set testRegex to nothing else I'd have jest bailing. Anyway, I'll remove this `test:unit`. At first I wanted to move all unit tests next to the sources in `src` folder to keep them well separated from all other "real-world" tests, but at the end it was already a big pr so I kept it like that :-D